themes/bootstrap3/templates/cart/cart.phtml

```diff
@@ -14,7 +14,7 @@
       <div class="checkbox pull-left flip">
```

Comment:
Removing "pull-left flip" from here would probably cause a wrap. But in case it doesn't...

Reply:
Removing pull-left flip doesn't seem to make any difference. How would you feel about actually adding an extra `<div class="pull-left flip">` around all of the buttons below the checkbox? This allows the checkbox and buttons to coexist when there is room, but it wraps all the buttons together when there is not enough room. The only disadvantage is it wraps the checkbox ABOVE the buttons, and I think we may have discussed a preference for wrapping the checkbox BELOW the buttons.

Reply:
Another option to consider: the existing search/bulk-action-buttons.phtml template wraps as-is, because it's coded as a checkbox div followed by a button ul. Would it make sense to re-code the cart buttons as a ul for consistency? Is there any reason not to do that? Again, this has the same issue of not wrapping the checkbox below the buttons, but maybe it makes sense to just retain current behavior and make the template code more consistent across the board.